src/commands/info.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
```

Comment:
New files must be written in Typescript. It's not a terribly large amount of work to adjust, but in doing so the code will be able to use `async` and `await` - the Typescript transpile process will change it into a form that is compatible with Node `6`.